common/src/main/java/org/wso2/testgrid/common/util/TestGridUtil.java

```diff
@@ -158,7 +160,14 @@ public static String executeCommand(String command, File workingDirectory, Deplo
      */
     public static String getTestGridHomePath() {
         String testGridHome = EnvironmentUtil.getSystemVariableValue(TESTGRID_HOME_ENV);
-        Path testGridHomePath = Paths.get(testGridHome);
+        Path testGridHomePath;
+        if (testGridHome == null) {
```

Comment:
Since we are doing a null check here, IMO we don't need to null-check the path again in Main#49.

Reply:
Done..